packages/flutter_tools/test/integration.shard/android_plugin_compilesdkversion_mismatch_test.dart

```diff
@@ -23,7 +23,8 @@ void main() {
     tryToDelete(tempDir);
   });
 
-  test('error logged when plugin Android compileSdkVersion higher than project', () async {
+  test('error logged when plugin Android compileSdkVersion higher than project',
```

Comment:
It looks like you ran `dart format` on this file, which should be undone. E.g., the breaks after `=` below violate flutter/flutter style (😒).

Reply:
Undone, I keep forgetting we have code that we are not allowed to auto format :/